core/src/main/java/tech/pegasys/web3signer/core/Eth1Runner.java

```diff
@@ -81,16 +80,10 @@ protected Router populateRouter(final Context context) {
 
   private ArtifactSignerProvider loadSigners(final Config config, final Vertx vertx) {
     final AzureKeyVaultSignerFactory azureFactory = new AzureKeyVaultSignerFactory();
-    final HashicorpConnectionFactory hashicorpConnectionFactory =
-        new HashicorpConnectionFactory(vertx);
 
     final Secp256k1ArtifactSignerFactory ethSecpArtifactSignerFactory =
         new Secp256k1ArtifactSignerFactory(
-            hashicorpConnectionFactory,
-            config.getKeyConfigPath(),
-            azureFactory,
-            EthSecpArtifactSigner::new,
-            true);
+            vertx, config.getKeyConfigPath(), azureFactory, EthSecpArtifactSigner::new, true);
```

Comment:
_prefer_ to see us passing in HashicorpFactory, and also the Interlock factory - as it means the lower layer doesn't need to be Vertx specific - but I can appreciate there maybe reasons it doesn't work so well.